docs/source-pytorch/accelerators/tpu_advanced.rst

```diff
@@ -12,11 +12,10 @@ Weight Tying/Sharing is a technique where in the module weights are shared among
 This is a common method to reduce memory consumption and is utilized in many State of the Art
 architectures today.
 
-PyTorch XLA requires these weights to be tied/shared after moving the model
-to the TPU device. To support this requirement Lightning provides a model hook which is
-called after the model is moved to the device. Any weights that require to be tied should
-be done in the `on_post_move_to_device` model hook. This will ensure that the weights
-among the modules are shared and not copied.
+PyTorch XLA requires these weights to be tied/shared after moving the model to the XLA device.
+To support this requirement, Lightning automatically finds these weights and ties them after
+the modules are moved to the XLA device under the hood. It will ensure that the weights among
+the modules are shared but not copied independently.
```

Comment:
@kaushikb11 Could you review this change in the doc page? My understanding is that #8555 suggested tying the weights without the need of user overriding `on_post_move_to_device`, and #9525 addressed the issue without updating the docs.